static/src/javascripts/projects/commercial/modules/dfp/prepare-googletag.js

```diff
@@ -31,13 +31,16 @@ const setDfpListeners = (): void => {
 
 const setPageTargeting = (): void => {
     const pubads = window.googletag.pubads();
-    const targeting = buildPageTargeting();
+    // because commercialFeatures has the potential to export itself as {}
```

Comment:
This is a question for @janua. His export of `commercialFeatures` is wrapped in a `try...catch` which may result in an empty object being exported, but under what conditions (other than an exception) I don't know.